src/app/_components/Header/Navigation/Navigation.tsx

```diff
@@ -13,37 +13,37 @@ const navigationItems: NavigationItemProps[] = [
         label: "Start"
     },
     {
-        href: "/tajemniczy",
-        label: "Dymowy Among"
+        href: "/download",
```

Comment:
Używajmy polskich nazw, np.:
```suggestion
        href: "/instalacja-tou",
```